python/lsst/ts/scheduler/driver/feature_scheduler_target.py

```diff
@@ -59,3 +59,8 @@ def __init__(
         )
         self.note = str(observation["note"][0])
         self.slewtime = float(observation["slewtime"][0])
+
+    @property
+    def rot(self) -> float:
+        """Return the physical rotator position in degrees."""
```

Comment:
Technically speaking you just return the value that is stored in the `observation` so you cannot say anything about the unit. But, yes, let's assume that the value always is in degrees.

Reply:
not really right? if you look, you will see that I am converting the value that is originally in radian to degrees.

Reply:
Oops totally missed that, thanks!